src/main/java/com/cloudogu/smeagol/wiki/infrastructure/PageResource.java

```diff
@@ -0,0 +1,30 @@
+package com.cloudogu.smeagol.wiki.infrastructure;
+
+import org.springframework.hateoas.ResourceSupport;
+
+public class PageResource extends ResourceSupport {
+
+    private String path;
+    private String content;
+
+    // TODO is commit embedded ???
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1135)